platform/preferred_languages.cpp

```diff
@@ -169,6 +172,41 @@ std::string GetCurrentNorm()
   return Normalize(GetCurrentOrig());
 }
 
+std::string GetUtf8MultilangSupported()
+{
+  std::string languageCode;
+
+  auto const & systemLanguages = GetSystemPreferred();
+  for (const auto & systemLanguage: systemLanguages)
+  {
+    const std::string normalizedLang = Normalize(systemLanguage);
+    if (StringUtf8Multilang::GetLangIndex(normalizedLang) != StringUtf8Multilang::kUnsupportedLanguageCode)
+    {
+      languageCode = normalizedLang;
+      break;
+    }
+  }
+
+  if (languageCode.empty())
+    throw std::runtime_error("Unsupported language.");
+
+  return languageCode;
```

Comment:
```suggestion
  return "en";
```

Reply:
If we want a default value, then [shoudn't that be](https://github.com/organicmaps/organicmaps/blob/master/coding/string_utf8_multilang.cpp#L14) `"default"`?

Reply:
As in `return StringUtf8Multilang::kDefaultCode;`